test/utilities/extensionUtils.ts

```diff
@@ -41,3 +42,78 @@ export async function findExtensionInRunningExtensionsList(
 
   return extensionWasFound;
 }
+
+export async function verifyAllExtensionsAreRunning(): Promise<void> {
```

Comment:
This verifyAllExtensionsAreRunning() function started out as a step in setup, but its use cases grew beyond setup to "Developer: Reload Window" and AnInitialSuite.  So it's now moved here to extensionUtils.ts per @CristiCanizales's suggestion.